airgun/entities/config_report.py

```diff
@@ -0,0 +1,69 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.utils import retry_navigation
+from airgun.views.config_report import ConfigReportDetailsView
+from airgun.views.config_report import ConfigReportsView
+
+
+class ConfigReportEntity(BaseEntity):
+    endpoint_path = '/config_reports'
+
+    def read(self, widget_names=None, host_name=None):
+        """Read all values for generated Config Reports"""
+        view = self.navigate_to(self, 'All')
+        view.search(host_name)
```

Comment:
Can you please add a similar check for host search as did in "export"?

Reply:
> Can you please add a similar check for host search as did in "export"?

Addressed